ui/src/app/pages/access/identity-help-text.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+
+import { AccessModule, Profile } from 'generated/fetch';
+
+import { Button } from 'app/components/buttons';
+import { FlexColumn, FlexRow } from 'app/components/flex';
+import {
+  getAccessModuleStatusByName,
+  isCompliant,
+} from 'app/utils/access-utils';
+import { openZendeskWidget } from 'app/utils/zendesk';
+import hhsLogo from 'assets/images/hhs-logo.png';
+import idMeLogo from 'assets/images/id-me-logo.svg';
+import loginGovLogo from 'assets/images/login-gov-logo.svg';
+
+import { styles } from './data-access-requirements';
+
+const ContactUs = (props: { profile: Profile }) => {
+  const {
+    profile: { givenName, familyName, username, contactEmail },
+  } = props;
+  return (
+    <div data-test-id='contact-us'>
+      <span
+        style={styles.link}
+        onClick={(e) => {
+          openZendeskWidget(givenName, familyName, username, contactEmail);
+          // prevents the enclosing Clickable's onClick() from triggering instead
+          e.stopPropagation();
+        }}
+      >
+        Contact us
+      </span>{' '}
+      if you’re having trouble completing this step.
+    </div>
+  );
+};
+
+export const IdentityHelpText = (props: {
+  profile: Profile;
+  afterInitialClick: boolean;
+  onClick?: Function;
+}) => {
+  const { profile, afterInitialClick, onClick } = props;
+
+  // don't return help text if complete or bypassed
+  const needsHelp = !isCompliant(
+    getAccessModuleStatusByName(profile, AccessModule.IDENTITY)
+  );
+
+  return (
+    needsHelp &&
+    (afterInitialClick ? (
+      <div style={styles.helpContainer}>
+        <div>
+          Looks like you still need to complete this action, please try again.
+        </div>
+        <ContactUs {...{ profile }} />
+      </div>
+    ) : (
+      <FlexColumn
+        style={{ ...styles.helpContainer, gap: '1rem', marginTop: '1rem' }}
+      >
```

Comment:
Minor: Could the `FlexColumn` be shared across the `afterInitialClick` logic?

Reply:
It will change the spacing of the "Looks like you still need to complete this action" section, but I think that that is alright. Updated.